pincer/client.py

```diff
@@ -200,24 +199,27 @@ def __init__(
         if isinstance(intents, Iterable):
             intents = sum(intents)
 
-        super().__init__(
-            token,
-            handlers={
-                # Gets triggered on all events
-                -1: self.payload_event_handler,
-                # Use this event handler for opcode 0.
-                0: self.event_handler,
-            },
-            intents=intents or Intents.all(),
-            reconnect=reconnect,
-        )
+        if intents is None:
+            intents = Intents.all()
```

Comment:
Any reason you switched from `intents = intents or Intents.all()`?

Reply:
`Intents.None` is 0 so it would be false.